tools/apigee-x-trial-provision/README.md

```diff
@@ -127,3 +127,180 @@ the `CERTIFICATES` environment variable:
+
+1. SSH into the Bastion VM instance-1. Press the SSH button.
+
+1. Login into your account to have necessary permissions.
+
+    ```sh
+    gcloud auth login --quiet
+    ```
+
+### Custom Network and Subnet Creation
+
+1. Define PROJECT variable by the ID of your project
+
+    ```sh
+    export PROJECT=<project-id>
+    ```
+
+1. Define environment variables that describe network topology
+
+    ```sh
+    export NETWORK=exco-vpc
+    export SUBNET=exco-vpc-dev-subnet
+
+    export REGION=us-central1
+    export ZONE=us-central1
+    export AX_REGION=us-central1
+
+    export NETWORK_CIDR=10.0.0.0/14
+    export SUBNET_CIDR=10.0.0.0/16
+    ```
+
+1. GCP: Create a VPC Network
+
+    ```sh
+    gcloud compute networks create $NETWORK \
+        --subnet-mode=custom \
+        --bgp-routing-mode=regional \
+        --mtu=1460
+    ```
+
+    Output:
+
+    ```sh
+    Created [https://www.googleapis.com/compute/v1/projects/qwiklabs-gcp-01-da1a8d6ae8b2/global/networks/exco-vpc].
+    NAME      SUBNET_MODE  BGP_ROUTING_MODE  IPV4_RANGE  GATEWAY_IPV4
+    exco-vpc  CUSTOM       REGIONAL
+    ...
+    ```
+
+1. Firewall Rules for internal, 22, and icmp traffic
+
+    ```sh
+    gcloud compute firewall-rules create $NETWORK-allow-internal --network $NETWORK --allow tcp,udp,icmp --source-ranges $NETWORK_CIDR
+    ```
+
+    Output:
+
+    ```sh
+    Creating firewall...⠹Created [https://www.googleapis.com/compute/v1/projects/qwiklabs-gcp-01-da1a8d6ae8b2/global/firewalls/exco-vpc-allow-internal].
+    Creating firewall...done.                                                                                                
+    NAME                     NETWORK   DIRECTION  PRIORITY  ALLOW         DENY  DISABLED
+    exco-vpc-allow-internal  exco-vpc  INGRESS    1000      tcp,udp,icmp        False
+    ```
+
+    ```sh
+    gcloud compute firewall-rules create fr-$NETWORK-ssh --network $NETWORK --allow tcp:22
+    ```
+
+    Output:
+
+    ```sh
+    Creating firewall...⠹Created [https://www.googleapis.com/compute/v1/projects/qwiklabs-gcp-01-da1a8d6ae8b2/global/firewalls/fr-exco-vpc-ssh].
+    Creating firewall...done.                                                                                                
+    NAME             NETWORK   DIRECTION  PRIORITY  ALLOW   DENY  DISABLED
+    fr-exco-vpc-ssh  exco-vpc  INGRESS    1000      tcp:22        False
+    ```
+
+1. GCP: Create a network subnet
+
+      ```sh
+      gcloud compute networks subnets create $SUBNET \
+          --network=$NETWORK \
+          --range=$SUBNET_CIDR \
+          --region=$REGION
+      ```
+
+      Output:
+
+      ```sh
+      Created [https://www.googleapis.com/compute/v1/projects/qwiklabs-gcp-01-da1a8d6ae8b2/regions/us-central1/subnetworks/exco-vpc-dev-subnet].
+      NAME                 REGION       NETWORK   RANGE
+      exco-vpc-dev-subnet  us-central1  exco-vpc  10.0.0.0/16
+      ```
+
+### Apigee X Org and GXLB
```

Comment:
Create Apigee X org and LB in custom network?

Reply:
fixed